build-parent/pom.xml

```diff
@@ -1281,5 +1279,39 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>user-session-settings</id>
+            <activation>
+                <file>
+                    <exists>${session.request.userSettingsFile.path}</exists>
+                </file>
+            </activation>
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <artifactId>maven-surefire-plugin</artifactId>
+                            <version>${version.surefire.plugin}</version>
+                            <configuration>
+                                <!-- combine.self suppresses warnings about java.io.tmpdir being defined twice -->
+                                <systemPropertyVariables combine.self="override">
```

Comment:
And it doesn't override anything that could be useful? It looks a bit weird to me?

Reply:
Instead of adding this to the global pom, I changed the code to distrust maven settings when coming from system properties.